app/src/ui/app.tsx

```diff
@@ -417,12 +417,14 @@ export class App extends React.Component<IAppProps, IAppState> {
     }
 
     const tip = state.state.branchesState.tip
+    const aheadBehind = state.state.aheadBehind
 
     if (tip.kind === TipState.Valid) {
       this.props.dispatcher.showPopup({
         type: PopupType.DeleteBranch,
         repository: state.repository,
         branch: tip.branch,
+        existsOnRemote: aheadBehind != null,
```

Comment:
Nit: `aheadBehind != null` -> `aheadBehind !== null`

You could also do the check where you declare `aheadBehind`

`const existsOnRemote = state.state.ahead !== null`

Reply:
How about just `!!aheadBehind`?

Reply:
Definitely shorter, but that `!!` though!

Reply:
Fixed in 46d78157e1c7747983f820470079ae0e43e8016e. 🔍